client/src/components/aggregations/ReportsMapWidget.tsx

```diff
@@ -35,8 +35,10 @@ const ReportsMapWidget = ({
     const markerArray = []
     values.forEach(report => {
       if (Location.hasCoordinates(report.location)) {
-        let label = _escape(report.intent || "<undefined>") // escape HTML in intent!
-        label += `<br/>@ <b>${_escape(report.location.name)}</b>` // escape HTML in locationName!
+        let label = report?.intent
+          ? `<a href="/reports/${report.uuid}" target="_blank">${_escape(report.intent)}</a>`
+          : "<undefined>"
```

Comment:
Better use `PAGE_URLS.REPORTS` from `pages/util` instead of hard-coding `/reports` here. Also `<undefined>` should be escaped as well, else it gets interpreted as an HTML tag… Anow now it doesn't link anywhere if the report intent is empty. Suggest to change this to:
```javascript
        let label = `<a href="${PAGE_URLS.REPORTS}/${report.uuid}" target="_blank">${_escape(report.intent || "<undefined>")}</a>` // escape HTML in intent!
```